bin/bazel_to_go.py

```diff
@@ -201,15 +201,16 @@ def template_protos(WKSPC):
         if file.endswith(".pb.go"):
             makelink(WKSPC + "/bazel-genfiles/pkg/adapter/template/" + file, WKSPC + "/pkg/adapter/template/" + file)
     for template in os.listdir(WKSPC + "/bazel-genfiles/template"):
-        for file in os.listdir(WKSPC + "/bazel-genfiles/template/" + template):
-            # check if there are files under /template/<some template dir>
-            if file.endswith("_tmpl.pb.go") or file.endswith("handler.gen.go"):
-                makelink(WKSPC + "/bazel-genfiles/template/" + template + "/" + file, WKSPC + "/template/" +template + "/" + file)
-            if os.path.isdir(WKSPC + "/bazel-genfiles/template/" + template + "/" + file):
-                for file2 in os.listdir(WKSPC + "/bazel-genfiles/template/" + template + "/" + file):
-                    # check if there are files under /template/<some uber dir>/<some template dir>
-                    if file2.endswith("_tmpl.pb.go") or file2.endswith("handler.gen.go"):
-                        makelink(WKSPC + "/bazel-genfiles/template/" + template + "/" + file + "/" + file2, WKSPC + "/template/" + template + "/" + file + "/" + file2)
+        if os.path.isdir(WKSPC + "/bazel-genfiles/template/" + template):
```

Comment:
aside: we really just need a script that finds all *.pb.go and *.gen.go files from $GENDIR (and not in vendor/) and symlinks them in right place to WKSPC.  We really shouldn't need to update this script every time.